integration_tests/CMakeLists.txt

```diff
@@ -675,12 +675,12 @@ RUN(NAME structs_32          LABELS cpython llvm c)
 RUN(NAME structs_33          LABELS cpython llvm c)
 RUN(NAME structs_34          LABELS cpython llvm c)
 
-RUN(NAME symbolics_01        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_02        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_03        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_04        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_05        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_06        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_01        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_02        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_03        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_04        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_05        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_06        LABELS cpython_sym c_sym)
```

Comment:
Do you plan to uncomment all of these?

Reply:
Yes , that should be done once #2255 ( The remaining implementation for this Pr) is ready !

Reply:
As discussed above, that should be done in subsequent PRs. This PR is ready, but we do not want to merge until we have fixes for the other tests in other PRs ready.

Reply:
> Do you plan to uncomment all of these?

These tests have now been addressed through #2255 . All tests pass as expected .